csrc/ir/printer.h

```diff
@@ -53,7 +53,6 @@ class TORCH_CUDA_CU_API IrTransformPrinter : public IrPrinter {
 
   void handle(Fusion* f) override;
 
- private:
```

Comment:
I don't see any reason why this has to be private. It is pretty convenient to just use this and print the transformation of a tensor.